### PR TITLE
PFS take over - Page 1

### PR DESCRIPTION
**Pull request by MarySnopok**

### CODE REVIEW

File: text_strings/client/sv.json [hunk 1]
```diff
@@ -2732,5 +2732,9 @@
   "shop_purchased_items_button": "Köpt",
   "general_user_account": "Ditt konto",
   "weekly_events_count": "%1$d händelser denna vecka",
-  "weekly_transactions_count": "%1$d transaktioner denna vecka"
+  "weekly_transactions_count": "%1$d transaktioner denna vecka",
+  "pfs_take_over_header_p1": "Gimi-kortet läggs ned",
+  "pfs_take_over_first_paragraph_p1": "Kortet som följt med din prenumeration av Gimi kommer att försvinna. Förändringen är ett steg i att göra appen mer fokuserad på finansiell utbildning för barn.",
+  "pfs_take_over_reduction_percent_banner_p1": "Vi sänker din prenumeration med %1$d%",
+  "pfs_take_over_second_paragraph_p1": "Som tack för att du är en trogen Gimi-användare sänker vi idag priset på din prenumeration från 49kr/mån till 29kr/mån."
```
Comment (gimi-anders) [inline text_strings/client/sv.json:2739]: I think it would be good to make the prices in this string dynamic.

File: text_strings/client/sv.json [hunk 2]
```diff
@@ -2732,5 +2732,9 @@
   "shop_purchased_items_button": "Köpt",
   "general_user_account": "Ditt konto",
   "weekly_events_count": "%1$d händelser denna vecka",
-  "weekly_transactions_count": "%1$d transaktioner denna vecka"
+  "weekly_transactions_count": "%1$d transaktioner denna vecka",
+  "pfs_take_over_header_p1": "Gimi-kortet läggs ned",
+  "pfs_take_over_first_paragraph_p1": "Kortet som följt med din prenumeration av Gimi kommer att försvinna. Förändringen är ett steg i att göra appen mer fokuserad på finansiell utbildning för barn.",
+  "pfs_take_over_reduction_percent_banner_p1": "Vi sänker din prenumeration med %1$d%",
```
Comment (gimi-anders) [inline text_strings/client/sv.json:2738]: If I remember correctly the percentage value should have a different background. To accomplish that you probably have to remove it from this string and manually add it in the code. Or do you have any other idea?

**Review by gimi-anders — CHANGES_REQUESTED** [review]

File: text_strings/client/sv.json (hunk 2 above)
Comment (MarySnopok) [reply text_strings/client/sv.json:2738]: @gimi-anders the whole string in this case has to be on a Teal alike background. I think you are thinking about page 3 , that has a percentage bubble with a pinkish background

Comment (gimi-anders) [reply text_strings/client/sv.json:2738]: Sorry, you are right, I was looking at an old design.

File: text_strings/client/sv.json [hunk 3]
```diff
@@ -2732,5 +2732,9 @@
...
+  "pfs_take_over_second_paragraph_p1": "Som tack för att du är en trogen Gimi-användare sänker vi idag priset på din prenumeration från %1$dkr/mån till %1$dkr/mån."
```
Comment (gimi-anders) [inline text_strings/client/sv.json:2739]: The currency needs to be within the dynamic value as well.
`"...från %1$d/mån till..."`

**Review by gimi-anders — CHANGES_REQUESTED** [review]

File: text_strings/client/sv.json (hunk 3 above)
Comment (MarySnopok) [reply text_strings/client/sv.json]: Oh sure those have to , thank you 🙏

**Review by gimi-anders — APPROVED** [review]

**Pull request merged**